docs/user/README.md

```diff
@@ -22,17 +24,19 @@ reaches a ready status, you can use the `--wait` flag and specify a timeout.
 To use `--wait` you must specify the units of the time to wait. For example, to
 wait for 30 seconds, do `--wait 30s`, for 5 minutes do `--wait 5m`, etc.
 
-**Note**: If you are running kind on MacOS or Windows then it is recommended
+**Note**: If you are running `kind` on MacOS or Windows then it is recommended
 that you have at least 4GB of RAM and disk space (these are estimates for a
```

Comment:
Thank you for mentioning this. I left it as a wild car back in the day because I wasn't sure how much disk space actually mattered for that specific issue.

Reply:
it's xx% disk space technically, unless we decide to remove disk eviction thresholds from the kubelet (which we might...)